integration/kube_integration_test.go

```diff
@@ -174,7 +174,7 @@ func testKubeExec(t *testing.T, suite *KubeSuite) {
 	username := suite.me.Username
 	kubeGroups := []string{kube.TestImpersonationGroup}
 	kubeUsers := []string{"alice@example.com"}
-	role, err := types.NewRoleV3("kubemaster", types.RoleSpecV6{
+	role, err := types.NewRole("kubemaster", types.RoleSpecV6{
```

Comment:
We skipped the Kube integration tests in our integration tests, but to get the pods, Rolev6 requires:

```go
role, err := types.NewRole("kubemaster", types.RoleSpecV6{
		Allow: types.RoleConditions{
			Logins:     []string{username},
			KubeGroups: kubeGroups,
                         KubernetesResources: []types.KubernetesResource{
                             {
                                 Kind: types.KindKubePod, Name: types.Wildcard, Namespace: types.Wildcard,
                              }
                         },


```

the same logic applies to every entry in this file.